actions/debootstrap_action.go

```diff
@@ -157,6 +157,26 @@ func (d *DebootstrapAction) RunSecondStage(context debos.DebosContext) error {
 	return err
 }
 
+func (d *DebootstrapAction) isLikelyOldSuite() bool {
+	// Guess if suite is something before usr-is-merged was introduced
```

Comment:
Can you move this to the function declaration so that go doc will work? e.g.

```suggestion
// Guess if suite is something before usr-is-merged was introduced
func (d *DebootstrapAction) isLikelyOldSuite() bool {
```